components/mindmap/menu-items/view.js

```diff
@@ -100,6 +100,18 @@ const PopperCard = ({ el, poppers }) => {
               <Row>{data.content}</Row>
             </>
           ) : null}
+          {data.audio ? (
+            <>
+              <Row>
+                <h5>Audio</h5>
+              </Row>
+              <Row>
+                <audio controls>
+                <source src="{data.audio}">
```

Comment:
```suggestion
                <source src="{data.audio}"/>
```

Reply:
Required to fix failed deployment: https://github.com/adityamukho/hivemind/pull/21#issuecomment-987766811